latest/index.bs

```diff
@@ -177,308 +177,99 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
+                |                # where the number of rows n MUST be the same across all arrays.
                 ├── .zgroup
-                |
-                ├── .zattrs       # `.zattrs` MUST contain `"encoding-type"`, which is set to `"dict"` by AnnData.           
-                │                 # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
-                │
-                ├── group         # You MAY add zarr groups.
-                |   |             # `uns` groups MAY contain groups, dataframes, dense arrays, and sparse arrays.
-                |   |
-                |   ├── .zgroup
-                |   |
-                |   ├── .zattrs   # `.zattrs` MUST contain `"encoding-type"`, which is set to `"csr_matrix"` by AnnData.           
-                |   │             # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
-                |   ...
-                |
-                ├── dataframe_0   # You MAY add dataframe group containers.
-                |   |                 # dataframes MAY be in the `uns` group or in a subgroup.
-                |   │
-                |   ├── .zgroup
-                |   │                     
-                |   ├── .zattrs       # `.zattrs` MUST contain `"_index"`, which is the name of the column in obs to be used as the index.           
-                |   │                 # `.zattrs` MUST contain `"column-order"`, which is a list of the order of the non-_index columns.
-                |   │                 # `.zattrs` MUST contain `"encoding-type"`, which is set to `"dataframe"` by AnnData.
-                |   │                 # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.2.0"` by AnnData.
-                |   │      
-                |   └── col_0         # Each column in the obs table is a 1D zarr array.
-                |       ├── .zarray   # Each columns MUST be chunked the same, but the chunking may be chosen by the user.
-                |       │
-                |       └─ 0
-                |
-                ├── dense_array       # You MAY dense arrays as n n-dimensional zarr arrays.
-                |   │                 # `dense_array` MUST not be a complex type (i.e., MUST be a single type)
-                |   │                 # `dense_array` MAY be chunked as the user desires.
-                |   |                 # `dense array` MAY be in the `uns` group or in a subgroup.
+                ├── .zattrs      # `.zattrs` MUST contain "column-order", which is a list of the order of the columns.
+                |                # `.zattrs` MUST contain "annotated-data", which is specified above.
+                |     
+                ├── column_1     # The table group MAY hold zero, one, or an arbitrary number of column arrays.
                 |   |
                 |   ├── .zarray
+                |   ├── .zattrs
```

Comment:
I didn't plan to store any special information in this file. So I guess it can probably be deleted, or is it required by the general Zarr layout for arrays?
This would also apply for the `.zattrs` file of the `row_names` array. Thanks for pointing this out!

Reply:
I think that if you have a `.zarray` then there's not usually a sibling `.zattrs` in zarr.

Reply:
Then I'll just delete these two files.